### PR TITLE
added new example grafana dashboard

### PR DESCRIPTION
**Pull request by malanmurphy**
Added a second example Grafana dashboard.

### CODE REVIEW

File: examples/grafana/README.md [hunk 1]
```diff
@@ -1,57 +1,44 @@
...
-The `NGINX Mesh Top` Grafana dashboard provides a high-level picture
-of the request volume and success rate of all your applications in the mesh.
-NGINX Service Mesh deploys Grafana and adds `NGINX Mesh Top` as the
-default dashboard. If you prefer to use an existing Grafana deployment, you can import
-the `NGINX Mesh Top` dashboard included in this directory. 
+Two Grafana dashboards have been included as examples for use in your environment: 
+- NGINX Mesh Top
+- NGINX Service Mesh Summary
+
+Both Grafana dashboards provides a high-level picture of the request volume and success rate of all your applications in the mesh, and each have been included to provide two different references for consuming and rendering NGINX Service Mesh metrics data.
```
Comment (kate-osborn) [inline examples/grafana/README.md:7]: "Both Grafana dashboards provide..."

File: examples/grafana/README.md [hunk 2]
```diff
@@ -1,57 +1,44 @@
...
-  Prometheus configuration.
- 
+  **Note:** If you are using the Prometheus server deployed by NGINX Service Mesh you can find the address by running [`nginx-meshctl config`](https://docs.nginx.com/nginx-service-mesh/reference/nginx-meshctl/#usage).
+  If you are using your own Prometheus server, make sure to add the NGINX Service Mesh [scrape config](../prometheus/README.md) to your Prometheus configuration.
 
-## Installing the NGINX Mesh Top Dashboard
+## Installing Example Dashboards
 
-To install the NGINX Mesh Top Dashboard using the Grafana UI follow these steps:
+To install the an example dashboard using the Grafana UI follow these steps:
```
Comment (kate-osborn) [inline examples/grafana/README.md:20]: "To install an example dashboard..."

File: examples/grafana/README.md [hunk 3]
```diff
@@ -1,57 +1,44 @@
...
 - Graphs:
-  - Request Volume: this graph shows the volume of requests per Pod, in
-    requests per second (`nginxplus_http_total_requests`).
-  - Pod Success:  this graph shows the percentage of requests that are
-    successful per Pod (`nginxplus_upstream_server_responses` with response codes of
-    `1xx` or `2xx`).
+  - Request Volume: this graph shows the volume of requests per Pod, in requests per second (`nginxplus_http_total_requests`).
+  - Pod Success:  this graph shows the percentage of requests that are successful per Pod (`nginxplus_upstream_server_responses` with response codes of `1xx` or `2xx`).
   - Sidecar Memory Usage (RSS): this graph shows the Resident Set Size (RSS) of the NSM sidecars.
   - Sidecar Memory Usage (Private): this graph shows the Private memory used by the NSM sidecars.
```
Comment (kate-osborn) [inline examples/grafana/README.md:40]: Should we outline the features of the summary dashboard as well?

File: examples/grafana/README.md [hunk 4]
```diff
@@ -1,57 +1,44 @@
...
-To install the NGINX Mesh Top Dashboard using the Grafana UI follow these steps:
+To install the an example dashboard using the Grafana UI follow these steps:
 - Select the New Dashboard button and click Import.
-- Upload the [nginx-mesh-top.json](nginx-mesh-top.json) file or copy and paste the contents of the file in the textbox and click Load.
-- Select the Prometheus data source you configured previously from the dropdown
-  menu and click Import.
+- Upload the dashboard JSON, ex: [nginx-mesh-top.json](nginx-mesh-top.json), file or copy and paste the contents of the file in the textbox and click Load.
+- Select the Prometheus data source you configured previously from the dropdown menu and click Import.
 
 ![nginx-mesh-top](dashboard.png)
```
Comment (kate-osborn) [inline examples/grafana/README.md:25]: Should we add an image of the summary dashboard here?

**Review by kate-osborn — CHANGES_REQUESTED** [review]

File: examples/grafana/README.md (hunk 1 above)
Comment (malanmurphy) [reply examples/grafana/README.md:7]: Will fix

File: examples/grafana/README.md (hunk 2 above)
Comment (malanmurphy) [reply examples/grafana/README.md:20]: Will fix

File: examples/grafana/README.md (hunk 3 above)
Comment (malanmurphy) [reply examples/grafana/README.md:40]: No, I'm fine just calling out the default as an example.

File: examples/grafana/README.md (hunk 4 above)
Comment (malanmurphy) [reply examples/grafana/README.md:25]: No, I'm fine just calling out the default as an example.

**Review by kate-osborn — APPROVED** [review]
LGTM! Just make sure to squash your commits when you merge 🙂

**Comment by malanmurphy** [comment]
Updates complete

**Pull request merged**